components/operator/api/v1alpha1/serverless_types.go

```diff
@@ -26,6 +27,13 @@ type DockerRegistry struct {
 	EnableInternal *bool `json:"enableInternal,omitempty"`
 	// Secret used for configuration of the Docker registry
 	SecretName *string `json:"secretName,omitempty"`
+	// PersistenceVolume configuration
+	PersistenceVolume *PersistenceVolume `json:"persistenceVolume,omitempty"`
```

Comment:
PersistentVolume or PersisnetVolumeClaim?
Persistent volume is only used here.

Reply:
maybe it would make sense to use simply `persistence`.
This way we are also inline with docker-registry conf [api](https://github.com/kyma-project/serverless/blob/main/config/serverless/charts/docker-registry/values.yaml#L51)

Reply:
https://sap-btp.slack.com/archives/G01KSPRNGLB/p1711548949775089